private/numerics/functions/bessel.rkt

```diff
@@ -0,0 +1,374 @@
+                       +.921566975526530895082307e+3
+                       +.74428389741411178824152e+2
+                       +.1e1))])
+          (* (sqrt (/ 2/π ax))
+             (- (* (cos ax-π/4) p₀)
+                (* z (sin ax-π/4) q₀)))))))
+
+
+(define (bessj₁ x)
+  (let ([ax (magnitude x)])
+    (if (< ax 8.0)
+        (let ([x² (* x x)])
+          (/ (* x
+                (poly-by-coeffs->value
+                 x²
+                 +.695364226329838502166085207e+8
+                 -.8356785487348914291918495672e+7
+                 +.3209027468853947029888682298e+6
+                 -.58787877666568200462723094e+4
+                 +.6121876997356943874446879769e+2
+                 -.3983107983952332023421699105e+0
+                 +.1705769264349617107854016566e-2
+                 -.4910599276555129440130592573e-5
+                 +.9382193365140744507653268479e-8
+                 -.1107352224453730633782671362e-10
+                 +.63194310317443161294700346e-14))
+             (poly-by-coeffs->value
+              x²
+              +.139072845265967685120764336e+9
+              +.6705346835482299302199750802e+6
+              +.1284593453966301898121332163e+4
+              +.1e+1)))
+        (let* ([z (/ 8.0 ax)]
+               [z² (* z z)]
+               [ax-3π/4 (- ax 3π/4)]
+               [p₁ (/ (poly-by-coeffs->value
+                       z²
+                       +.1290918471896188077350689e+5
+                       +.1309042051103506486292571e+5
+                       +.313275295635506951011069e+4
+                       +.17431379748379024599685e+3
+                       +.122850537643590432633e+1)
+                      (poly-by-coeffs->value
+                       z²
+                       +.1290918471896187879332737e+5
+                       +.1306678308784402036110575e+5
+                       +.310928141677002883350924e+4
+                       +.16904721775008609992033e+3
+                       +.1e+1))]
+               [q₁ (/ (poly-by-coeffs->value
+                       z²
+                       ;This line or the next is in error
+                       ;+.14465282874995208675225e+3   
+                       +.14465282874995208765225e+3
+                       +.1744291689092425885102e+3
+                       +.5173653281836591636536e+2
+                       +.379944537969806734901e+1
+                       +.36363466476034710809e-1)
+                      (poly-by-coeffs->value
+                       z²
+                       +.308592701333231723110639e+4
+                       +.373434010601630179517765e+4
+                       +.11191098527047487025919e+4
+                       +.8522392064341340397334e+2
+                       +.1e+1))]
+               [ans (* (sqrt (/ 2/π ax))
+                       (- (* (cos ax-3π/4) p₁)
+                          (* z (sin ax-3π/4) q₁)))])
+          (if (< x 0.0) (- ans) ans)))))
+
+(define (bessj n x)
```

Comment:
Something's wrong. When I make graphs using:
```
  (plot
   (for/list ([i (in-range 1 6)])
     (function (lambda (x) (bessj i x)) #:color i #:label (format "y = bess_~a" i)))
   #:x-min 0 #:x-max 10
   #:out-file "yolo.png" #:out-kind 'png
   )
```

I get this:
![bessj](https://cloud.githubusercontent.com/assets/1731829/8679945/01aff828-2a2c-11e5-8155-31991a7cc93f.png)

Compare to http://mathworld.wolfram.com/BesselFunctionoftheFirstKind.html
(Which is what the `scmutils` version looks like)